OpenSim/Simulation/Manager/Manager.h

```diff
@@ -71,7 +71,7 @@ class OSIMSIMULATION_API Manager
     Storage *_stateStore;
 
    int _steps;
-   /** Number of integration step trys. */
+   /** Number of integration step tries. */
```

Comment:
This is inconsistent with the name of the corresponding variable but I think it's still an improvement.